scripts/release/prepare.js

```diff
@@ -0,0 +1,302 @@
+#!/usr/bin/env node
```

Comment:
same comment as in the next script. Please implement this as a shell script, the functionality here is mostly trivial, it does not require 300-line long script